src/defaults/Autocomplete.tsx

```diff
@@ -0,0 +1,160 @@
+import { SearchKeyword, SearchProduct } from '../api/search'
+
+interface AutocompleteProps {
+    response?: {
+        keywords: {
+            hits: SearchKeyword[]
+        }
+        products: {
+            hits: SearchProduct[]
+        }
+    }
+    history?: {
+        item: string
+    }[]
+}
+
+export function Autocomplete({ response, history }: AutocompleteProps) {
+    const hasKeywords = !!response?.keywords?.hits?.length
+    const hasProducts = !!response?.products?.hits?.length
+    const hasHistory = !!history?.length
+
+    if (!hasKeywords && !hasProducts && !hasHistory) {
+        return null
+    }
+
+    return (
+        <div className="ns-autocomplete-results">
```

Comment:
should we parametrize the class names in these components?

Reply:
Good idea. Same as we want to do in search templates.